src/network/networkpacket.h

```diff
@@ -118,6 +119,32 @@ class NetworkPacket
 	NetworkPacket &operator>>(video::SColor &dst);
 	NetworkPacket &operator<<(video::SColor src);
 
+	template<typename T> NetworkPacket &operator >> (std::vector<T> &dst)
+	{
+		u16 count;
```

Comment:
i think we really should use u32, 2 bytes for a such thing is nothing, breaking protocol because we send 80k elements is :)

Reply:
Lists longer than 65k entries will be rare, but I will extend it just for the case.

Reply:
C99 ensures that `size_t` must be at least a `u16`.
http://irc.minetest.net/minetest-dev/2018-08-21#i_5382824